media/css/m24/vars/_lib.scss

```diff
@@ -5,10 +5,6 @@
 @forward  '~@mozilla-protocol/core/protocol/css/includes/lib';
 @forward 'animation';
 @forward 'color';
-@forward 'fonts';
 @forward 'grid';
 @forward 'spacing';
 @forward 'text';
-
-
-$mq-max: '(min-width: 1440px)';
```

Comment:
💟 🚀 

This is also in:
https://github.com/mozilla/bedrock/blob/70c975b621240e36f8acfe052646a44609789811/media/css/privacy/privacy-firefox.scss#L14
and:
https://github.com/mozilla/bedrock/blob/70c975b621240e36f8acfe052646a44609789811/media/css/legal/terms-firefox.scss#L14
(and perhaps tangentially?:)
https://github.com/mozilla/bedrock/blob/70c975b621240e36f8acfe052646a44609789811/media/css/m24/springboard.scss#L166